src/main/java/org/commcare/formplayer/aspects/MetricsAspect.java

```diff
@@ -173,12 +191,32 @@ private long getSubmitBlockedTime() {
         return submitService.getSubmitTimer().durationInSeconds();
     }
 
-    private void sendTimingWarningToSentry(SimpleTimer timer) {
+    private Map<String, Long> getSlowRequestThresholds() {
```

Comment:
It probably doesn't matter since this is only called once, but it might be nice if this map were part of the object rather than calculated on demand

Reply:
Moved to constructor.